src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs

```diff
@@ -764,6 +764,36 @@ public Task<string> GetLastCommitShaAsync(string repoUri, string branch)
             return GetLastCommitShaAsync(owner, repo, branch);
         }
 
+        /// <summary>
+        ///     Get the commits in a repo
+        /// </summary>
+        /// <param name="repoUri">Repository uri</param>
+        /// <returns>Return all the commits. Null if no commits were found.</returns>
```

Comment:
Outadeted comment. I usually deletes param and returs whet it is obvious which mostly is.